testspec.yml

```diff
@@ -13,9 +13,9 @@ phases:
       - docker run --name pg -d -p 5432:5432 postgres
       - docker run --name test -d -e RAILS_ENV=test --link pg:pg temp_image:test /bin/bash -c "tail -f /dev/null"
       - docker exec test bundle install --with test --retry 3 --jobs 20
-      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && export ELASTICSEARCH_URL='http://es:9200' && rake db:create"
-      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && export ELASTICSEARCH_URL='http://es:9200' && rake db:setup"
-      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && export ELASTICSEARCH_URL='http://es:9200' && rake"
+      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && rake db:create"
+      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && rake db:setup"
+      - docker exec test /bin/bash -c "export DATABASE_URL='postgres://postgres@pg:5432/test?template=template0&pool=5&encoding=unicode' && API_ROOT='https://ccs.api/' && rake"
```

Comment:
Shouldn't that be `&& export API_ROOT=...` ?

Reply:
Ah yep I'll add `export`

`rake db:create` and `rake db:setup` don't rely on `API_ROOT` being set afaik, I think it was just failing on one of the tests without it being set

Reply:
:+1: